lib/kafka/datadog.rb

```diff
@@ -247,6 +249,7 @@ def buffer_overflow(event)
       def drop_messages(event)
         tags = {
           client: event.payload.fetch(:client_id),
+          topic: event.payload.fetch(:topic),
```

Comment:
This event doesn't currently include the topic, since that would entail introspecting the buffer. I'd skip it for now, or update this part of the code to figure out the per-topic counts and instrument an event for each topic:

https://github.com/zendesk/ruby-kafka/blob/5d46d9eba3b7dd7852ed6d02c3c6e8b648d06db3/lib/kafka/async_producer.rb#L210-L212

Reply:
I'll just remove this for now -- I was mainly interested in recording the topic for `enqueue_message`